templates/disable-inactive-freeipa-users.j2.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Disable inactive FreeIPA users.
+
+[Service]
```

Comment:
Do we want to alphabetize these `[Unit]` and `[Service]` blocks?  I'm fine with it as is, just figured I'd stoke the conversation here.

Reply:
The current order is the standard for `systemd` services.  `systemd` timers, services, sockets, etc. are all units, and a service is a particular type of unit.  So the sections are in order from most general to most specific.

Reply:
Makes sense, thanks for the info. 👍